share/p2p/shrexnd/server.go

```diff
@@ -140,19 +143,19 @@ func validateRequest(req pb.GetSharesByNamespaceRequest) error {
 }
 
 // respondNotFoundError sends internal error response to client
-func (srv *Server) respondNotFoundError(stream network.Stream) {
+func (srv *Server) respondNotFoundError(logger *zap.SugaredLogger, stream network.Stream) {
```

Comment:
why

Reply:
wait i still don't understand why it needs to be passed @walldiss

Reply:
I want to use logger with fields down the stack. Do you mean it could be done differently?